examples/index.html

```diff
@@ -26,28 +36,49 @@
         setRequestHeader(name, value) {
           // Do nothing.
         }
-        getResponseHeader(name, value) {
-          // Do nothing.
+        getResponseHeader(name) {
+          if (name === 'Content-Type' && focusedInput.classList.contains('json')) {
+            return 'application/json'
+          }
         }
         send() {
-          this.status = input.value === '422' ? 422 : 200
+          if (focusedInput.classList.contains('json')) {
+            this.responseText = '{"text": "success", "html": "<b>success</b>"}'
+          } else {
+            this.responseText = 'success'
+          }
+          this.status = focusedInput.value === '422' ? 422 : 200
```

Comment:
Return example error message to be shown:

<img width="277" alt="image" src="https://user-images.githubusercontent.com/1153134/61911827-940c2800-af06-11e9-8866-410dff9576c1.png">


```js

          const json = focusedInput.classList.contains('json')
          if (focusedInput.value === '422') {
            this.status = 422
            this.responseText = json ? '{"text": "bad", "html": "<b>bad</b>"}' : 'bad'
          } else {
            this.status = 200
            this.responseText = json ? '{"text": "success", "html": "<b>success</b>"}' : 'success'
          }
```